drivers/usb/udc/udc_numaker.c

```diff
@@ -1216,12 +1216,22 @@ static void numaker_usbd_isr(const struct device *dev)
 
 	uint32_t volatile usbd_intsts = base->INTSTS;
 	uint32_t volatile usbd_bus_state = base->ATTR;
+	uint32_t usbd_inten_all =
```

Comment:
I see 'volatile' above, please remove them in a separate commit.

Reply:
Removed `volatile`